ui/ad_position.js

```diff
@@ -91,7 +91,7 @@ shaka.ui.AdPosition = class extends shaka.ui.Element {
    */
   setPosition_() {
     const adsInAdPod = this.ad.getSequenceLength();
-    if (adsInAdPod > 1) {
+    if (adsInAdPod > 1 && this.ad.isLinear()) {
       // If it's a single ad, showing 'Ad 1 of 1' isn't helpful.
```

Comment:
Also, I would add a line to the end of this comment explaining why we only show this text for linear ads.

Reply:
Done!